README.md

```diff
@@ -183,18 +199,18 @@ supporters on our BountySource. Thank you for your support!
 This project is licensed under `Apache-2.0`. Read [`LICENSE`](https://github.com/SethMichaelLarson/trytravis/blob/master/LICENSE) for more information.
 
 ```
- Copyright 2017 Seth Michael Larson
+Copyright 2017 Seth Michael Larson
```

Comment:
This change is nice but not necessary for Travis v3 API

Reply:
ok